docs/datasources/softlayer_ssh_key.md

```diff
@@ -0,0 +1,41 @@
+# `softlayer_ssh_key`
+
+Use this data source to import the details of an *existing* SSH key as a read-only data source.
```

Comment:
I would add that you can use this to get the SSH key id that you could use in other resources like virtual_guest.